generate_tests.js

```
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+// Auto-generate Mocha/Chai tests comparing custom klsh commands to real Bash
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)